src/app/pages/dynamic-stepper/dynamic-stepper.component.html

```diff
@@ -15,7 +15,7 @@
         <mat-icon>delete</mat-icon>
     </button>
 </mat-toolbar>
-<div *ngIf="!procedureFinished">
+<div *ngIf="!procedureFinished" class="stepper-container">
```

Comment:
Is this class doing anything? I don't see it referenced anywhere else

Reply:
This is now removed :)